pkgs/development/python-modules/tkinter-gl/default.nix

```diff
@@ -0,0 +1,37 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  pythonOlder,
+  setuptools-scm,
+  tkinter,
+}:
+
+buildPythonPackage rec {
+  pname = "tkinter-gl";
+  version = "1.0";
+  pyproject = true;
+
+  disabled = pythonOlder "3.11";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "tkinter_gl";
+    sha256 = "sha256-gZgR9ABiMMxOm7mtDZRXV1WyFMS1DrNCel91QcdJgrs=";
+  };
+
+  build-system = [ setuptools-scm ];
+
+  propagatedBuildInputs = [ tkinter ];
+
+  doCheck = true;
```

Comment:
No need to set this due to
https://github.com/NixOS/nixpkgs/blob/7c90d7bf2fc6d61a5deab2ff8542f5ffe9697bb4/pkgs/development/interpreters/python/mk-python-derivation.nix#L389